assignment/assignment.hs

```diff
@@ -40,8 +44,11 @@ toDigits num = "replace me"
 -- Write a recursive function that counts the number of times
 -- a given value occurs in a list
 
-count :: (Eq a, Num p) => a -> [a] -> p
-count element list = 0 -- replace me
+--count :: (Eq a, Num p) => a -> [a] -> p
+count element [] = 0
+count element (x:xs) = if (element==x) then 1+count element xs else count element xs
+
+
 
```

Comment:
Again, a personal preference, but I like things a little more spread out: I try to have each line of code do one or maybe two things:

~~~ hs
count val [] = 0 
count val (x:xs) =
    n + count val xs
    where 
        n = if val == x then 1 else 0
~~~